doc/en/components/grids/_shared/export-excel.md

```diff
@@ -0,0 +1,234 @@
+```
+
+You may access the exporter service by defining an argument of type [`ExcelExporterService`]({environment:demosBaseUrl}/classes/excelexporterservice.html) in the component's constructor and the {Platform} framework will provide an instance of the service. To export some data in MS Excel format you need to invoke the exporter service's [`export`]({environment:demosBaseUrl}/classes/excelexporterservice.html#export) method and pass the {ComponentTitle} component as first argument.
+
+Here is the code which will execute the export process in the component's razor file:
+
+```razor
+// component.ts
+import { ExcelExporterService, ExcelExporterOptions } from 'igniteui-{Platform}';
+import { {Component} } from 'igniteui-{Platform}';
+
+@ViewChild('{ComponentTitle}') public {ComponentTitle}: {ComponentTitle}Component;
+
+constructor(private excelExportService: ExcelExporterService) {
+}
+
+public exportButtonHandler() {
+  this.excelExportService.export(this.{ComponentTitle}, new ExcelExporterOptions('ExportedDataFile'));
+}
+```
+
+If all went well, you should see the {ComponentTitle} component and a button under it. When pressing the button, it will trigger the export process and the browser will download a file named "ExportedDataFile.xlsx" which contains the data from the {Component} component in MS Excel format.
+
+## Export All Data
+
+There are some cases when you might be using remote operations like *Paging* and the Grid won't have access to all of its data. In these cases, we recommend using the [Excel Export Service](../exporter-excel.md) and pass the whole data collection, if available. Example:
+
+```ts
+public exportButtonHandler() {
+  this.excelExportService.exportData(this.localData, new ExcelExporterOptions('ExportedDataFile'));
+}
+```
+
+<!-- ComponentStart: Grid -->
+## Export Grouped Data
+
+To export grouped data you just need to group the {Component} by one or more columns. The browser will download a file named "ExportedDataFile.xlsx" which contains the data from the {Component} component in MS Excel format grouped by the selected column. Example:
+
+
+<code-view style="height: 800px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz" 
+           iframe-src="{environment:demosBaseUrl}/services/export-excel-sample-1" alt="{Platform} Grouped Data Excel Exporter Example">
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+## Export Multi Column Headers Grid
+
+It is now possible to export {Component} with defined [multi-column headers](multi-column-headers.md). All headers will be reflected in the exported excel file as they are displayed in the {Component}. If you want to exclude the defined multi-column headers from the exported data you can set the [exporter option]({environment:demosBaseUrl}/classes/exporteroptionsbase.html) [ignoreMultiColumnHeaders]({environment:demosBaseUrl}/classes/exporteroptionsbase.html#ignoremulticolumnheaders) to `true`.
```

Comment:
remove all hard-code links and just use API members (in camelcase) and wrap them in backticks, e.g. ``` `ExcelExporterService` ``` ``` `Export` ```